pkgs/applications/audio/master_me/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, pkg-config
+, python3, Cocoa }:
+
+stdenv.mkDerivation rec {
+  pname = "master_me";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "trummerschlunk";
+    repo = "master_me";
+    rev = version;
+    sha256 = "sha256-FG3X1dOF9KRHHSnd5/zP+GrYCB2O0y+tnI5/l9tNhyE=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libGL libX11 libXext libXrandr ]
+                ++ lib.optionals stdenv.isDarwin [ Cocoa ];
+
+  postPatch = ''
+    patchShebangs ./dpf/utils/
+    substituteInPlace ./dpf/utils/res2c.py \
+      --replace '/usr/bin/env python3' ${python3.interpreter}
```

Comment:
patchShebangs is not working if it can't find python in any of the inputs

Reply:
this is indeed a shebang https://github.com/DISTRHO/DPF/blob/d9088872a3ee65d33c6b9a60222dc9364b22b81a/utils/res2c.py

Reply:
this is still open and not good practise